package.json

```diff
@@ -10,6 +10,7 @@
     "prop-types": "^15.8.1",
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229075575-71823a38-b6cd-4c79-ad7d-6d1e685f2581.png)

- [ ] The sections which are shown in the image above should have at least the following commands ⬇️ to make your readme file descriptive 👍

### Install

```
npm install
```

### Usage

```
npm start
```

### Run tests

```
npm test
```

### Deployment

```
npm run build
```